CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,107 @@
+ReproHack Project Code of Conduct
+==============================
+
+This code of conduct outlines our expectations for participants within the
+ReproHack community, as well as steps to reporting unacceptable behavior. We are
+committed to providing a welcoming and inspiring community for all and expect
+our code of conduct to be honored. Anyone who violates this code of conduct may
+be banned from the community.
```

Comment:
Maybe online and offline??

Reply:
Good call. In fact I'll adapt something in the Carpentries CoC:

> Everyone participating in The Carpentries activities is required to conform to the Code of Conduct. This Code of Conduct applies to all spaces managed by The Carpentries including, but not limited to, workshops, email lists, and online forums such as GitHub, Slack and Twitter. Workshop hosts are expected to assist with the enforcement of the Code of Conduct.

Reply:
Yes, great. That should cover most online fora/spaces

Reply:
Done!